server/render/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import ReactDomServer from 'react-dom/server';
+import Helmet from 'react-helmet';
+import memoize from 'lodash/function/memoize';
+import superagent from 'superagent';
+import LruMap from 'lru-map';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const memoizedRenderToString = memoize( ReactDomServer.renderToString, element => JSON.stringify( element ) );
```

Comment:
Let's assume that any data retrieval has to happen outside of the `renderToSting` call, because there will always be some async component to it. Does that we are caching a call here that actually does very little, giving us an extra layer of cache alongside the data cache that may not save any time?